fastify-session-authentication/package.json

```diff
@@ -10,14 +10,14 @@
   "author": "Denis Fäcke",
   "license": "MIT",
   "dependencies": {
-    "fastify": "^2.3.0",
-    "fastify-cookie": "^3.0.2",
-    "fastify-formbody": "^3.1.0",
-    "fastify-session": "^2.1.0"
+    "@fastify/cookie": "^7.2.0",
+    "@fastify/formbody": "^7.0.1",
+    "@fastify/session": "^9.0.0",
+    "fastify": "^4.2.1"
   },
   "devDependencies": {
-    "fastify-cli": "^1.0.0",
-    "request": "^2.88.0",
-    "tap": "^14.1.7"
+    "fastify-cli": "^4.3.0",
+    "request": "^2.88.2",
```

Comment:
Maybe we want to drop request too (possibly in a follow-up PR)

Reply:
Good idea this PR is still open so will add to here